ui/src/components/ImageAutocomplete.tsx

```diff
@@ -0,0 +1,28 @@
+import Autocomplete from "@mui/material/Autocomplete/Autocomplete";
+import TextField from "@mui/material/TextField/TextField";
+import { useGetImages } from "../hooks/useGetImages";
+
+interface Props {
+  value: string;
+  onChange(v: string): void;
+}
+
+export const ImageAutocomplete = ({ value, onChange }: Props) => {
+  const { data: images } = useGetImages();
+  const imageNames = (images || []).reduce((acc, image) => {
+    acc = acc.concat(image.Repotags || [])
```

Comment:
I'm afraid with the last change the images are not listed: 

![image](https://user-images.githubusercontent.com/15997951/182820376-228a34e8-0a2e-48c9-8fcf-b0ff103c7b2a.png)